src/apidoc.rs

```diff
@@ -0,0 +1,116 @@
+// Copyright 2023 MOSEC Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+use std::{collections::BTreeMap, str::FromStr};
+
+use serde::Deserialize;
+use utoipa::openapi::{
+    path::Operation, request_body::RequestBody, Components, OpenApi, PathItemType, RefOr, Response,
+    Responses, Schema,
+};
+
+#[derive(Deserialize, Default)]
+pub(crate) struct PythonApiDoc {
```

Comment:
```suggestion
pub(crate) struct PythonAPIDoc {
```

Align with the naming convention in this project.